src/app/static/src/app/components/adr/SelectRelease.vue

```diff
@@ -159,10 +173,14 @@
             valid() {
                 this.$emit("valid", this.valid);
             },
+            releases(){
+                this.preSelectRelease();
```

Comment:
You should be able to fix coverage by checking method is called if releases changes